pkg/apb/executor.go

```diff
@@ -86,18 +88,18 @@ func ExecuteApb(
 		// of uniquenes and will meet DNS name requirements.
 		executionContext.Namespace = uuid.New()
 		executionContext.Targets = append(executionContext.Targets, context.Namespace)
-		//Creating project
-		_, err := openshitftClient.CreateProject(executionContext.Namespace)
+		// Create namespace.
+		namespace := v1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Labels: map[string]string{transientNameSpaceKey: spec.FQName},
```

Comment:
@enj creating namespaces and adding the metadata about which APB started the namespace.